domain/schema/controller.go

```diff
@@ -308,3 +312,158 @@ BEGIN
 END;
 `[1:]
 }
+
+func modelMigrationSchema() string {
+	return `
+CREATE TABLE model_migration (
```

Comment:
I think we should actually use the existing `external_controller` table to record data common between this table and that one. Just have the controller's UUID here and include a FK constraint.

We'll have to adjust the migration logic slightly when we get to that part, because there is an explicit step at the end to copy this the data between the migration collection and the external controllers collection.